pkg/pillar/hypervisor/hypervisor.go

```diff
@@ -238,3 +240,30 @@ func PCISameControllerGeneric(id1 string, id2 string) bool {
 
 	return tag1 == tag2
 }
+
+func LaunchSwtpmAndWait(id string, seconds int) (string, error) {
```

Comment:
```suggestion
func launchSwtpmAndWait(id string, seconds int) (string, error) {
```

it is only called from the same package, isn' it?

Reply:
didn't realize that kvm is in the hyper package.